cmd/run/cmd.go

```diff
@@ -280,4 +287,6 @@ func init() {
 	Cmd.Flags().StringVar(&cfg.ProfilerHost, "profiler-host", "localhost", "Host for the Profiler server")
 	Cmd.Flags().IntVar(&cfg.ProfilerPort, "profiler-port", 6060, "Port for the Profiler server")
 	Cmd.Flags().StringVar(&txStateValidation, "tx-state-validation", "tx-seal", "Sets the transaction validation mechanism. It can validate using the local state index, or wait for the outer Flow transaction to seal. Available values ('local-index' / 'tx-seal'), defaults to 'tx-seal'.")
+	Cmd.Flags().Uint64Var(&cfg.TxRequestLimit, "tx-request-limit", 1, "Number of transaction submissions to allow per the specified interval. Applies only on Testnet.")
+	Cmd.Flags().StringVar(&txDurationLimit, "tx-duration-limit", "3s", "Time interval upon which to enforce transaction submission rate limiting. Applies only on Testnet.")
```

Comment:
you could use `DurationVar` which will automatically parse the value into a `time.Duration`

the current naming sounds like a limit for the duration, not the other way around. Also, this applies on whichever network the gateway is configured to use
```suggestion
	Cmd.Flags().StringVar(&txDurationLimit, "tx-request-limit-duration", "3s", "Time interval upon which to enforce transaction submission rate limiting.")
```

Reply:
Good point :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/750/commits/b33cd9c3f139358b5e93785476769e6d31fd5b98